src/main/scala/tilelink/Broadcast.scala

```diff
@@ -314,6 +317,15 @@ class TLBroadcastTracker(id: Int, lineBytes: Int, probeCountBits: Int, bufferles
   io.out_a.bits.mask    := o_data.bits.mask
   io.out_a.bits.data    := o_data.bits.data
   io.out_a.bits.corrupt := Bool(false)
+  io.out_a.bits.user.lift(AMBAProt).foreach { x =>
```

Comment:
could these be passed through from `io.in_a`?

Reply:
Maybe they could could, but that is not what was done in the closed source example I was working of off, for reasons I am personally unaware of. Perhaps @terpstra @jsmithsf or @daveparry can comment about the particular selection of bits and the decision to ignore any settings supplied by inward request initiators?